hw/bsp/ch32v307/Link.ld

```diff
@@ -0,0 +1,170 @@
+ENTRY( _start )
+
+__stack_size = 4096;
+
+PROVIDE( _stack_size = __stack_size );
+
+
+MEMORY
+{
+	FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 288K
+	RAM (xrw) : ORIGIN = 0x20000000, LENGTH = 32K
+}
+
+
+SECTIONS
+{
+
+	.init :
+	{
+		_sinit = .;
+		. = ALIGN(4);
+		KEEP(*(SORT_NONE(.init)))
+		. = ALIGN(4);
+		_einit = .;
+	} >FLASH AT>FLASH
```

Comment:
why the `AT>FLASH` here?

Reply:
I'm not sure. I think it's redundant because both sections are the same.
Apart from the change to the stack size, needed for examples to run, this is just as supplied by the vendor examples.

Reply:
I see, yeah it's redundant here, hence the question